### DISCUSSION TITLE
Feature request: don't take over youtube links, add a youtube link to media viewer

### DISCUSSION THREAD

**Discussion by notuntoward** [aidenlx/media-extended#326] [Q&A]
This is the best youtube annotator I've seen on Obsidian, but after using it a while, I have a couple requests:

1.  Don't always send youtube links clicks to this plugin's media viewer window

When I click on a youtube link, the media window always opens up instead of the youtube on the web browser.  I'd most often want to instead go directly to the browser, because it's bigger, and because I can block ads there (when this plugin's media viewer opens up, it's a least a minute before the ads finish playing).  

For this reason alone, I've had to disable this plugin.  I only re-enable it when I want to take youtube notes.  I didn't need to disable [Timestamp Notes plugin](https://github.com/juliang22/ObsidianTimestampNotes), for example

2. Add a timestamped youtube link in the media viewer.  

For the reasons above, viewing videos in the browser is better.  I'd like to get there directly from the media viewer, at the timestamp shown on the viewer.  A "show in browser" button, or something like that, would be handy.

**Comment by aidenlx** [comment]
There are ad block built-in for YouTube, could you check which version of plugin you're using?

For window size issue, simply drag tab with player out to dedicated window will give you freedom to control video size.

If you do want to open url in browser, you can right click on link to select "open in browser"

**Reply by notuntoward** [reply]
I'm using v 3.1.0 of the plugin 

- but at least for me, adverts aren't blocked.  For example, on this  [video](https://www.youtube.com/watch?v=Eic-SmIMG2Y), there are 2 adverts before the video I'm interested in starts.
- thanks for the tip on moving the media extended video to a new window.
- ah, now I see the right click menu item, `open link in default browser`.  That's a good workaround, but for right clicks, I'd much rather see `open link in media extended` -- and for normal clicks, that the video opens with the default browser.   I _annotate_ a video link maybe once a week, but _watch_ a video link maybe once a day. I'd guess this is true for most users, and that most users would be surprised to see an override of the default clicking behavior.  It seems to me that [Timestamp Notes plugin](https://github.com/juliang22/ObsidianTimestampNotes) handles this more intuitively,
